cli/cmd/testdata/install_single_namespace_output.golden

```diff
@@ -1,6 +1,12 @@
+
```

Comment:
is this whitespace intentional?

Reply:
it's expected but not ideal. subsequent PRs will remove single namespace template wrapping, and introduce templatizing via stage, we can clean this up then.